app/controllers/declaration/AdditionalFiscalReferencesController.scala

```diff
@@ -40,83 +41,104 @@ class AdditionalFiscalReferencesController @Inject()(
   journeyType: JourneyAction,
   errorHandler: ErrorHandler,
   customsCacheService: CustomsCacheService,
+  exportsCacheService: ExportsCacheService,
   mcc: MessagesControllerComponents
 )(implicit appConfig: AppConfig, ec: ExecutionContext)
-    extends FrontendController(mcc) with I18nSupport {
+    extends {
+  val cacheService = exportsCacheService
```

Comment:
FYI You can just have `override val  cacheService: ExportsCacheService` as a constructor param. I think a side effect of using the extends block here is you you can reference your `ExportsCacheService` using the variable name `exportsCacheService` *and* `cacheService` which might be a little confusing

Reply:
this is the approach we have taken across our caching work. Any changes to this we should cover in an overall discussion and change all controllers if we decide there is a better way of doing this. I.e. get rid of the modelcacheable trait and add the functions it has directly into the exportsCacheService as one suggestion.